src/mudlet-lua/lua/DB.lua

```diff
@@ -441,6 +441,8 @@ function db:create(db_name, sheets, force)
     db.__env = luasql.sqlite3()
   end
 
+  assert(db_name == db:safe_name(db_name), "Santized name does not match argument.  Check you are using the correct database.")
```

Comment:
How would a database newbie know how to fix this error? Can we word it better?

Reply:
Yeah, true.. hrm..

`Database name contains illegal characters.  Only alphanumeric characters are valid.  Database not created or loaded.`

Reply:
I think that would work great.